README.md

```diff
@@ -160,21 +160,18 @@ A temporary `.download` file is kept in the output directory. If `annie` is run
 
 Cookies can be provided to `annie` with the `-c` option if they are required for accessing the video.
 
-**Note:**
+Cookies can be a string or a text file.
 
-* cookies must match the following format:
```

Comment:
This format needs to be retained.

I prefer this:

**Note: cookies must match the following format:**

```
name=value; name2=value2; ...
```

Cookies can be a string or a text file.

As a string:
...